apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -39,60 +26,88 @@ type Story = StoryObj<typeof Tooltip>;
 
 export const TopPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.TOP,
+    placement: TooltipPlacement.Top,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const BottomPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.BOTTOM,
+    placement: TooltipPlacement.Bottom,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const LeftPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.LEFT,
+    placement: TooltipPlacement.Left,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const RightPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.RIGHT,
+    placement: TooltipPlacement.Right,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const DefaultMaxTooltipWidth: Story = {
-  args: {
-    trigger: <>Tooltip Trigger</>,
-    children: (
-      <>
-        This is the default max tooltip width. You can customize the max width
-        if desired. See the "Custom Max Tooltip Width" story for an example of
-        setting a custom max width.
-      </>
-    ),
-    placement: TooltipPlacement.BOTTOM,
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>
+          This is the default max tooltip width. You can customize the max width
+          if desired. See the "Custom Max Tooltip Width" story for an example of
+          setting a custom max width.
+        </span>
+      </Tooltip>
+    );
   },
 };
 
 export const CustomMaxTooltipWidth: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: (
-      <>
-        This is a customized max tooltip width. You can customize the max width
-        if desired.
-      </>
-    ),
-    placement: TooltipPlacement.BOTTOM,
```

Comment:
We should make each story as "pure" for communicating a given idea as possible. The idea of a custom tooltip width doesn't need to add any complexity of also specifying a custom placement.

Therefore removed a custom `placement` here.